packages/contracts/test/token/erc20/governance-wrapped-erc20.ts

```diff
@@ -36,12 +38,13 @@ describe('GovernanceWrappedERC20', function () {
   let signers: SignerWithAddress[];
   let governanceToken: GovernanceWrappedERC20;
   let erc20: TestERC20;
+  let dao: DAO;
```

Comment:
This DAO is not used in the tests. Please revert.

Reply:
https://github.com/aragon/osx/pull/596/commits/6710f4efa85095b3a2451c25180643479940cb66